src/stores/spaces/SpaceStore.ts

```diff
@@ -553,6 +567,9 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
             this.roomIdsBySpace.set(MetaSpace.Orphans, new Set(orphans.map(r => r.roomId)));
         }
 
+        // bust cache for meta spaces
+        enabledMetaSpaces.forEach(metaSpaceKey => this._aggregatedSpaceCache.roomIdsBySpace.delete(metaSpaceKey));
```

Comment:
Why are we even bothering to aggregate-cache the metaspaces? They always contain no nesting so don't need duplicating in a cache